src/martialarts.cpp

```diff
@@ -471,8 +471,8 @@ bool ma_buff::is_valid_player( const player &u ) const
 
 void ma_buff::apply_player( player &u ) const
 {
-    u.dodges_left += dodges_bonus;
-    u.blocks_left += blocks_bonus;
+    u.set_num_dodges_bonus(u.get_num_dodges_bonus() + dodges_bonus);
```

Comment:
```suggestion
    u.set_num_dodges_bonus( u.get_num_dodges_bonus() + dodges_bonus );
```

Reply:
I need to figure out how to put astyle into my visual studio, don't I.

Reply:
See https://github.com/CleverRaven/Cataclysm-DDA/pull/24006 for instruction.

Reply:
Thanks! I think I've gotten that working.